aecore_eqc/aec_keys_mock.erl

```diff
@@ -0,0 +1,7 @@
+-module(aec_keys_mock).
+
+-compile(export_all).
+-compile(nowarn_export_all).
+
+peer_pubkey() ->
```

Comment:
I see, you  want to mock the aec_keys:peer_pubkey to always return this fixed value and you do this via a fallback in QuickCheck mocking. That seems fine.